### PR TITLE
[4.0] Remove the deprecated code in the MultilangstatusHelper class

### PR DESCRIPTION
**Pull request by laoneo**
Remove the deprecated code in the MultilangstatusHelper class.

### CODE REVIEW

File: administrator/components/com_languages/Helper/MultilangstatusHelper.php [hunk 1]
```diff
@@ -83,56 +83,6 @@ public static function getContentlangs()
...
-	}
-
-	/**
-	 * Method to return a list of language home page menu items.
-	 *
-	 * @return  array of menu objects.
-	 *
-	 * @deprecated  4.0  Use Multilanguage::getSiteHomePages() instead.
-	 */
-	public static function getHomepages()
```
Comment (wilsonge) [inline administrator/components/com_languages/Helper/MultilangstatusHelper.php:118]: This is still being used by the com_languages admin view

**Review by wilsonge — CHANGES_REQUESTED** [review]
See comment

**Review by wilsonge — APPROVED** [review]
I can't read my IDE properly

**Comment by laoneo** [comment]
Do we need to document deprecated code removal in core extensions? I would say that the API is enough.

**Comment by wilsonge** [comment]
Yeah we don't need to document code removal in core extensions (we do need to note any architectural changes - but not function level changes like this)

**Pull request merged**